packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  box-sizing: border-box;
+  color: $text-body;
+  font-family: $body-font-family;
```

Comment:
Since we've removed `font: inherit` for `button` and `input`, we'll want to add `font-variant: normal` to this list. Since `font-feature-settings` appears later, that will take precedence where supported.

With that, all browser-default style normalizations for the `font` group should be covered here.

Without `inherit`, we can't guarantee that some properties won't be overridden by element-specific browser styles, but they'll be hard to miss.